#852 - SassResourceReference does not work if SCSS files are in JARs

reckart opened the pull request:
If the root SCSS resource is not on the file system, then load its content from its URL and compile it as a String. This is useful for compiling SCSS resources localed in JAR files. Local imports are not supported in such SCSS files, but they can usually be replaced with package imports.

reckart committed:
#852 - SassResourceReference does not work if SCSS files are in JARs

- If the root SCSS resource is not on the file system, then load its content from its URL and compile it as a String. This is useful for compiling SCSS resources localed in JAR files. Local imports are not supported in such SCSS files, but they can usually be replaced with package imports.

chrisco484 reviewed (commented):
Is scssConcent a typo? Did you mean scssContent?

reckart commented:
indeed

reckart committed:
#852 - SassResourceReference does not work if SCSS files are in JARs

- Fixed typo in variable name

reckart committed:
#852 - SassResourceReference does not work if SCSS files are in JARs

- The classpath location can be found after the last JAR_SCHEME occurence. This also works in situations where there are multiple nested JARs (e.g Spring Boot FAT JARs).

martin-g commented:
Thank you, @reckart !
